### ISSUE TITLE
Could not resolve all files for configuration ':application:classpath'. (How to control the sequence of tasks in a Gradle multiproject)

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/inspectus4katalon-gradle-plugin#10]
at 0.1.5, the .workflows/tests.yml ran and `gradle test` failed.

```
A problem occurred configuring project ':application'.
> Could not resolve all files for configuration ':application:classpath'.
   > Could not find com.kazurayam:inspectus4katalon:0.1.5.
     Searched in the following locations:
       - file:/home/runner/.m2/repository/com/kazurayam/inspectus4katalon/0.1.5/inspectus4katalon-0.1.5.pom
       - https://plugins.gradle.org/m2/com/kazurayam/inspectus4katalon/0.1.5/inspectus4katalon-0.1.5.pom
     Required by:
         project :application
```

the settings.gradle files was:
```
rootProject.name = 'inspectus4katalon'
include(':development', ':application')
```

This means, the project is a Gradle multi-project which contains 2 subprojects: `:application` and `:developmet`. When the `gradle test` task ran on the CI server, the `:application:test` task ran before `:development:test`. At that timing, the jar of inspectus3katalon-0.1.5.jar has not present. Therefore the test failed.

I think, I need to control the sequence of subprojects tasks. `gradle test` should execute `gradle :development:test` first, and then `gradle :application:test` next. But how can I do it?

**Comment by kazurayam** [comment]
https://stackoverflow.com/questions/51376806/depend-on-multiple-gradle-tasks-in-multi-project-build

**Comment by kazurayam** [comment]
On my local PC, I could reproduce this problem.

(1) I edited `<projectDir>/settings.gradle` to be
```
rootProject.name = 'inspectus4katalon'
include(':development', ':application')
```

(2) I removed the inspectus4katalon-xxx.jar in the mavenLocal repository. 
```
$ cd ~
$ cd .m2/repository/com/kazurayam
$ rm -rf inspectus4katalon
```

(3) I executed the test
```
$ cd <projectDir>
$ gradle test
```

Then I got:
```
$ gradle test

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':application'.
> Could not resolve all files for configuration ':application:classpath'.
   > Could not find com.kazurayam:inspectus4katalon:0.1.5.
     Searched in the following locations:
       - file:/Users/kazuakiurayama/.m2/repository/com/kazurayam/inspectus4katalon/0.1.5/inspectus4katalon-0.1.5.pom
       - https://plugins.gradle.org/m2/com/kazurayam/inspectus4katalon/0.1.5/inspectus4katalon-0.1.5.pom
     Required by:
         project :application

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s

```

**Comment by kazurayam** [comment]
1. The :application:test task depends on the :development:publishToMavenLocal task.
2. The :development:publishToMavenLocal task depends on the :development:build task.
3. The :development:build task depends on the test task
4.  --> loop back to the 1 

This is difficult to solve.

**Comment by kazurayam** [comment]
At the moment, the best fix is not to include `:application` in the settings.gradle file.

```
rootProject.name = 'inspectus4katalon'
include(':development')
```

This makes least trobles.

**Comment by kazurayam** [comment]
resolved by #20
